.storybook/custom/components/DocsPage.js

```diff
@@ -40,14 +40,42 @@ const DocsPage = () => {
         document.querySelectorAll('.toc-link').forEach( x => x.setAttribute('target','_self'));
     }, []);
 
+    const showImport = () => {
+        let groups = context.kind.split('/');
+        const name = groups[1];
+        return (
+            <div className='docs-single-import'>
+                <Import componentName={name} />
+            </div>
+        );
+    }
+
+    const showSubImports = () => {
+        const subComps = context?.parameters?.subcomponents;
+        const subImports = [];
+        if(subComps){
+            const compNames = Object.keys(subComps);
+            compNames?.forEach((name, index) => {
+                subImports.push(<Import key={index} componentName={subComps[name]?.displayName}/>)
+            })
+        }
+        return (
+            <div className='docs-multi-imports'>
+                {subImports}
+            </div>
+        );
+    };
+
     return (
         <>
         <Header />
         <Title />
         <Toc />
         <Subtitle />
+        {context?.parameters?.deprecated && <Description desc={context?.parameters?.deprecated} />}
+        {!context?.parameters?.noImport && showImport()}
+        {context?.parameters?.displaySubComponentImports && showSubImports()}
         {context?.parameters?.description && <Description desc={context?.parameters?.description} />}
```

Comment:
Not caused by this pr (it's in master too) - but our descriptions are gone 😢

Reply:
I can see the description for our Tree component here: https://deploy-preview-1172--fundamental-react.netlify.app/?path=/docs/component-api-tree--wrapping-and-non-wrapping-contents

Reply:
I got the story descriptions working here, but not the component descriptions: https://github.com/SAP/fundamental-react/pull/1189